package.json

```diff
@@ -34,8 +34,21 @@
     "@types/etag": "^1.8.1",
     "@types/fs-extra": "^9.0.13",
     "@types/node-fetch": "^2.6.1",
+    "ava": "^4.3.1",
+    "esbuild-node-loader": "^0.8.0",
     "eslint": "8.22.0",
     "jiti": "^1.13.0",
     "typescript": "^4.6.2"
+  },
+  "ava": {
+    "extensions": {
+      "ts": "module"
+    },
+    "nodeArguments": [
+      "--loader=esbuild-node-loader"
+    ],
+    "files": [
+      "tests/**/*.test.ts"
```

Comment:
I prefer colocating tests, but I notice we typically have a tests folder for tests so went with the same format.